projects/15_basic_material/main.cpp

```diff
@@ -106,7 +106,10 @@ class ProjApp
     grfx::MeshPtr              mSphere;
     grfx::MeshPtr              mCube;
     grfx::MeshPtr              mMonkey;
-    grfx::MeshPtr              mAltimeterModel;
+    grfx::MeshPtr              mMeasuringTape;
```

Comment:
Are those needed, as the pointer is stored in the vector anyway? They are only read during loading, but then stored into the vector.

Reply:
These aren't needed.  I've removed in the latest push and added some logic to pull the vertex binding information.